src/smartpeak/source/core/SequenceSegmentProcessors/CalculateCalibration.cpp

```diff
@@ -37,6 +37,16 @@
 
 namespace SmartPeak
 {
+  std::set<std::string> CalculateCalibration::getInputs() const
+  {
+    return { "Features"};
```

Comment:
"Standards Concentrations" and "Quantitation Methods" should also be inputs I believe.

Reply:
indeed. thanks